src/webgl/shaders/light.vert

```diff
@@ -1,78 +1,69 @@
+precision mediump float;
+
+// vertex attributes
 attribute vec3 aPosition;
 attribute vec3 aNormal;
 attribute vec2 aTexCoord;
 
+// matrices
 uniform mat4 uViewMatrix;
 uniform mat4 uModelViewMatrix;
 uniform mat4 uProjectionMatrix;
 uniform mat3 uNormalMatrix;
+
+// ambient lights
 uniform int uAmbientLightCount;
+uniform vec3 uAmbientLightColor[8];
+
+// directional lights
 uniform int uDirectionalLightCount;
-uniform int uPointLightCount;
+uniform vec3 uDirectionalLightDirection[8];
```

Comment:
Just a note that when this merges, we'll need to update the shader uniform and attribute documentation here: https://github.com/processing/p5.js/wiki/WebGL-Module-Architecture